### ISSUE TITLE
Unable to run CLI setup

### ISSUE THREAD

**Issue by DavidWells** [semantic-release/semantic-release#245]
I am getting a CLI setup error:

![image](https://cloud.githubusercontent.com/assets/532272/16321065/c847c2e4-394f-11e6-8dc0-8641331961be.png)

```
? What CI are you using? Travis CI Pro
ERR! semantic-release Could not create Travis CI hook.
ERR! semantic-release Error: Travis encountered an error, sorry :(
ERR! semantic-release     at /Users/davidwells/.nvm/versions/node/v4.2.4/lib/node_modules/semantic-release-cli/node_modules/travis-ci/lib/travis-ci.js:49:30
ERR! semantic-release     at Request._callback (/Users/davidwells/.nvm/versions/node/v4.2.4/lib/node_modules/semantic-release-cli/node_modules/travis-ci/lib/travis-http.js:49:13)
ERR! semantic-release     at Request.self.callback (/Users/davidwells/.nvm/versions/node/v4.2.4/lib/node_modules/semantic-release-cli/node_modules/travis-ci/node_modules/request/index.js:148:22)
ERR! semantic-release     at emitTwo (events.js:87:13)
ERR! semantic-release     at Request.emit (events.js:172:7)
ERR! semantic-release     at Request.<anonymous> (/Users/davidwells/.nvm/versions/node/v4.2.4/lib/node_modules/semantic-release-cli/node_modules/travis-ci/node_modules/request/index.js:886:14)
ERR! semantic-release     at emitOne (events.js:82:20)
ERR! semantic-release     at Request.emit (events.js:169:7)
ERR! semantic-release     at IncomingMessage.<anonymous> (/Users/davidwells/.nvm/versions/node/v4.2.4/lib/node_modules/semantic-release-cli/node_modules/travis-ci/node_modules/request/index.js:837:12)
ERR! semantic-release     at emitNone (events.js:72:20)
ERR! semantic-release  [Error: Travis encountered an error, sorry :(]
```

I'm also not able to test out or run `semantic-release pre` and get a `semantic-release ERR! pre Failed to determine new version.` error no matter what and the output buffer isn't readable
![image](https://cloud.githubusercontent.com/assets/532272/16321105/f67adc82-394f-11e6-9e66-e27c06240fbe.png)

This is a larger project with lots of git history. Is it possible to enable semantic release on a repo with tons of history/releases?

Thanks

**Comment by davemachado** [comment]
This might be tied back to the Travis CI API bug from a few weeks ago.

**Comment by marvindanig** [comment]
Did any of you get it working finally? On my rather new/empty nodejs project travis-ci shows`build:success` but no releases are published on Github or on npm. Any help would be great!

**Comment by ByteDefault** [comment]
I'm in the same situation, I've tried with a fresh install but no luck. Any link to the Travis CI bug to track the progress?

**Comment by davemachado** [comment]
I went through the closed issues list and didn't see anything that referenced this being fixed. If someone else would like to look through the issues and look for duplications, that would be great.

**Comment by terales** [comment]
Don't know real solution, but you can init semantic release with selecting `Other CI` option, so it will print `GH_TOKEN` and `NPM_TOKEN` which you should add to the Travis CI → Settings → Environment Variables.

After the first push it should work as expected.

**Comment by ByteDefault** [comment]
@terales I've tried manual configuration and it doesn't work for private repos because semantic-release tries to connect with travis-ci.org instead of travis-ci.com (which is the server for private repos).

Is there any way to set travis-ci.com as default server?

**Comment by terales** [comment]
@ByteDefault file a new issue with logs from Travis. This issue is about automatic setup.

**Comment by terales** [comment]
@DavidWells, @davemachado, @marvindanig  

Issue with a buffer size was fixed in PR #15 and released in [6.3.1](https://github.com/semantic-release/semantic-release/releases/tag/v6.3.1).

But `npm` installs 4.3.5 by default, so you need to specify version manually:
```sh
npm i -S semantic-release@6.3.1
```

**Comment by boennemann** [comment]
Both of the problems are fixed with the latest versions of semantic-release (6.3.2) and the cli (3.0.0).
